### ISSUE TITLE
 Algonquin college registration problem

### ISSUE THREAD

**Issue by Hilaire-Cedrick** [Hilaire-Cedrick/first-website#1]
Mr Thomas Bradley,
Please how can i do to enroll at Algonquin College. I have registered myself on learntheweb.course to follow your courses, but seems something is missing since i cant select the courses and see my progresses from my profile.
With the hope to ha a favorable return, accept my sympaty and greetings.
@thomasjbradley

**Comment by thomasjbradley** [comment]
I don’t believe Algonquin has any running courses using Learn the Web right now. But you can still follow along, but going into the course you’re doing & it will show you your progress.

**Comment by Hilaire-Cedrick** [comment]
Hi Sir,
Thanks very much for your reply Sir, will keep learning and I make goods
used of all these great courses  you have put in our disposal .
Thanks
